src/render/program/fill_extrusion_program.js

```diff
@@ -24,7 +24,8 @@ export type FillExtrusionUniformsType = {|
     'u_matrix': UniformMatrix4f,
     'u_lightpos': Uniform3f,
     'u_lightintensity': Uniform1f,
-    'u_lightcolor': Uniform3f
+    'u_lightcolor': Uniform3f,
+    'u_vertical_gradient': Uniform1f
```

Comment:
this is just 1 or 0 right? in that case we can use an integer uniform `Uniform1i` to follow the convention of other uniforms used as booleans such as `u_is_zoom_constant` for symbol icons/sdfs

Reply:
This actually needs to be a float because the calculation in the shader has float constants.